server-openjdk.yaml

```diff
@@ -0,0 +1,94 @@
+- name: native-builder
+  version: 1.0
+  from: quay.io/quarkus/centos-quarkus-maven:20.0.0-java11
+  description: Builder image for native config generator
+  artifacts:
+  - name: config-generator-src
+    url: https://github.com/infinispan/infinispan-image-artifacts/archive/2.0.0.Dev01.tar.gz
+    target: config-generator-src.tar.gz
+  modules:
+    repositories:
+    - path: modules
+    install:
+    - name: quarkus.config-generator
+
+- name: infinispan/server
+  version: 11.0.0-SNAPSHOT
+  description: Infinispan Server
+  from: registry.access.redhat.com/ubi8/ubi-minimal:8.2-267
+  artifacts:
+  - name: config-generator
+    image: native-builder
+    path: /opt/config-generator
+  - name: server
+    url: https://downloads.jboss.org/infinispan/11.0.0.Dev04/infinispan-server-11.0.0.Dev04.zip
```

Comment:
Can't we use some form of variable substitution to avoid hardcoding the version number in many places ?

Reply:
Cekit doesn't allow variable substitution in the yaml, instead it allows yaml nodes to be [overridden](https://docs.cekit.io/en/latest/handbook/overrides.html).

However the creation of these URLs are automated by `Jenkinsfile-release` anyway, so it's simply case of entering the Infinispan version in the build and the descriptor file is updated accordingly. The `Jenkinsfile-release` hasn't been updated to reflect the fact we have multiple images, but that's my plan for today.